webclient/src/index.html

```diff
@@ -1,10 +1,27 @@
 <!DOCTYPE html>
-<html lang="${{_lang_}}$">
+<html lang="${{_lang_}}$" prefix="og: https://ogp.me/ns#">
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
-    
+
+    <meta name="description" content="Navigate around TUM with excellence. This is a website to find rooms, buildings and other places at TUM easier than the current RoomFinders.">
+    <meta name="author" content="@octycs and @CommanderStorm">
+    <meta name="copyright" content="GNU General Public License v3.0. Some images have other licences.">
+    <meta name="robots" content="index, follow">
+    <meta name="rating" content="safe for kids">
+
+    <meta name="twitter:card" content="summary_large_image">
+    <meta property="og:type" content="website"/>
+    <meta property="og:title" content="NavigaTUM" />
+    <meta property="og:url" content="https://roomapi.tum.sexy">
+    <meta property="og:description" content="Navigate around TUM with excellence. This is a website to find rooms, buildings and other places at TUM easier than the current RoomFinders." />
+    <meta property="og:image" content="<!-- @echo app_prefix -->assets/logo-card.png" />
+    <meta property="og:image:width" content="1200" />
+    <meta property="og:image:height" content="630" />
+    <meta property="og:image:mime" content="image/png" />
+    <meta property="og:site_name" content="TUM.sexy" />
```

Comment:
```suggestion
    <meta property="og:site_name" content="NavigaTUM" />
```

I think this makes more sense on a social link card.

Reply:
[ogp.me](https://ogp.me/) states:
> og:site_name - If your object is part of a larger web site, the name which should be displayed for the overall site. e.g., "IMDb".

I think, that TUM.sexy does fit this discription more, or do you think differently? I dont have a strong opinion on this